### DISCUSSION TITLE
Enable screensaver

### DISCUSSION THREAD

**Discussion by sebycarta** [adrg/libvlc-go#93] [Q&A] [answered]
Hi all,
i developed a simple program to play a video from webcam. I noticed that when the program is active, the screensaver is disabled ( screen always on).
How i can disable this features? i didn't find any method to enable screensaver.
Thank you.

**Comment by adrg** [comment, marked as answer]
Hi @sebycarta,

You can try adding the `--no-disable-screensaver` option when you are initializing the library.

```go
if err := vlc.Init("--no-disable-screensaver"); err != nil {
	log.Fatal(err)
}
defer vlc.Release()
```

**Reply by sebycarta** [reply]
thank you very much, it works!
